2/homework/rnn_training.ipynb

```diff
@@ -0,0 +1,736 @@
+{
```

Comment:
To much code in public space without functions for my taste but visualisations look great and results are neat